include/mqt-core/dd/Node.hpp

```diff
@@ -39,31 +39,30 @@ struct mNode {                        // NOLINT(readability-identifier-naming)
   RefCount ref{};                     // reference count
   Qubit v{};                          // variable index
   std::uint8_t flags = 0;
-  // 32 = unused (was used to mark a node with is symmetric).
-  // 16 = marks a node resembling identity
+  // 32 = marks a node with is symmetric.
+  // 16 = free (used to mark identity nodes)
```

Comment:
```suggestion
  // 32 = unused (was used to mark a node which is symmetric)
  // 16 = unused (was used to mark a node resembling the identity)
```
seems a little more consistent to me